### DISCUSSION TITLE
Using MFA with Windows Remote Desktop

### DISCUSSION THREAD

**Discussion by saif27076** [neos-sdi/adfsmfa#264] [Q&A] [answered]
Hi Guys, Any idea or work around of using MFA with RDP

**Comment by redhook62** [comment, marked as answer]
There is no particular problem when you are in an RDP session, 
You just need an available network connection on the machine in hand.

**Reply by saif27076** [reply]
i mean using mfa for authentication when you connect to the server

**Reply by redhook62** [reply]
Sorry, you can watch here : https://learn.microsoft.com/en-us/windows-server/identity/ad-fs/operations/access-control-policies-in-ad-fs
Indeed, the MFA does not decide "to whom the MFA applies" but only "how to apply the MFA"
You can create a new access control Policy as indicated in the Microsoft documentation. and, assign it to your relying parties

**For administrators in RDP**
- Intranet source or specific network
- For specified groups...
- Require MFA

**Internet source**
- Extranet
- Everyone
- Require MFA

**Intranet source**
- Intranet
- everyone

regards
